developer/README.md

```diff
@@ -0,0 +1,21 @@
+# Dependencies (TEMPORARY)
```

Comment:
Why Temporary?

Reply:
Just leaving this here for reference - I will need to re-write the dev docs.